src/crxRegistry/README.md

```diff
@@ -0,0 +1,9 @@
+# ContributionRewardExt Rewarder Registry
```

Comment:
I am not in principle against having such a registrty, but it seems that at this point (we are implementing this pattern just a single time, for the first time) it is premature - too early to generalize (or to know how to gneeralize this).

Reply:
yeah, and too early to be commenting on it in the PR  :-)